skiko/src/commonMain/kotlin/org/jetbrains/skiko/SkikoApp.kt

```diff
@@ -0,0 +1,54 @@
+package org.jetbrains.skiko
+
+import org.jetbrains.skia.Canvas
+
+interface SkikoApp {
+    // Input
+    fun onKeyboardEvent(event: SkikoKeyboardEvent)
```

Comment:
This is tricky question, as it generally presumes certain external event system introp. Guess we better have it as method in event instead, if we'll need it. And it requires design.

Reply:
> method in event instead

`consume` method is fine, I guess. It will be similar to AWT, web and Compose itself (don't know about other platforms). But we should be aware that it will make event mutable.

> if we'll need it

We'll probably need this. SkiaLayer can be used not only in windows, but in separate views (ComposePanel, for example), where we should know if the view has event consumed (for example, scroll on panel should be consumed, if the panel itself can be scrolled)

> And it requires design.

I guess, if we don't switch to this API for CfD 1.0, it is fine to make breaking changes in this interface.